src/components/header/PanelHeader.tsx

```diff
@@ -1,49 +1,45 @@
 /** @jsxImportSource @emotion/react */
-import { css } from '@emotion/react';
+import styled from '@emotion/styled';
 import type { ReactNode } from 'react';
 
 import { Button } from '../button/index.js';
 import { SelectedTotal } from '../selected-total/index.js';
 
-const styles = {
-  container: css({
-    display: 'flex',
-    flexDirection: 'row',
-    alignItems: 'center',
-    justifyContent: 'space-between',
-    borderBottom: '0.55px solid rgb(240, 240, 240)',
-  }),
-  leftContainer: css({
-    display: 'flex',
-    flexDirection: 'row',
-    flex: 1,
-    '& > button': {
-      padding: '0 5px',
-      minWidth: 'auto',
-    },
-  }),
-};
-
 interface PanelHeaderProps {
   total?: number;
   current?: number;
   onClickSettings?: () => void;
   children?: ReactNode;
 }
 
-export function PanelHeader({
-  total,
-  current,
-  children,
-  onClickSettings,
-}: PanelHeaderProps) {
+const PanelContainer = styled.div`
```

Comment:
I think this name is confusing because we are not in a Panel component. `PanelHeaderContainer` or `HeaderContainer` or just `Container` would be better, no?
Same below.

Reply:
I renamed vars to HeaderContainer & HeaderLeftContainer is this ok @targos ?

Reply:
yes